firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift

```diff
@@ -55,12 +55,8 @@ extension BrowserViewController {
                                      method: .press,
                                      object: .keyCommand,
                                      extras: ["action": "reload"])
-
-        guard let tab = tabManager.selectedTab else { return }
-
-        if !contentContainer.hasHomepage {
-            tab.reload()
-        }
+        store.dispatch(GeneralBrowserAction(windowUUID: windowUUID,
```

Comment:
Dispatching the action should only happen if the feature flag for toolbar refactor is turned on.